modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaGetFileMethodCheck.java

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.source.formatter.check;
+
+import com.liferay.petra.string.StringBundler;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.util.StringUtil;
+import com.liferay.source.formatter.check.util.JavaSourceUtil;
+import com.liferay.source.formatter.check.util.SourceUtil;
+
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * @author Tamyris Bernardo
+ */
+public class UpgradeJavaGetFileMethodCheck extends BaseFileCheck {
+
+	@Override
+	protected String doProcess(
+			String fileName, String absolutePath, String content)
+		throws Exception {
+
+		List<String> importNames = JavaSourceUtil.getImportNames(content);
+
+		if (!importNames.contains(
+				"com.liferay.document.library.kernel.service." +
+					"DLFileEntryLocalServiceUtil") ||
+			!importNames.contains(
+				"com.liferay.document.library.kernel.service." +
+					"DLFileEntryLocalService")) {
```

Comment:
This check is essentially dealing with two cases (`DLFileEntryLocalServiceUtil` and `DLFileEntryLocalService`). But with this conditional, this check only works on Java files if they have both cases in them. Shouldn't it be when there are either cases and deal with each one accordingly?

For example, if the Java file only had the import `DLFileEntryLocalService`, this check would skip this file and the `DLFileEntryLocalService` case would not be handled.